deployer/migrations/0006_delete_outdated_resources.sql

```diff
@@ -0,0 +1,5 @@
+-- delete any old records of these so that they are not sent to r-r
+DELETE FROM resources WHERE type = "static_folder";
+DELETE FROM resources WHERE type = "turso";
+DELETE FROM resources WHERE type = "metadata";
+DELETE FROM resources WHERE type = "custom";
```

Comment:
Nit: this could be one line, `DELETE FROM resources WHERE type IN ("static_folder", ...)`.